iceoryx_hoofs/test/moduletests/test_index_queue.cpp

```diff
@@ -32,21 +32,9 @@ class IndexQueueTest : public ::testing::Test
     using index_t = typename Queue::value_t;
 
   protected:
-    IndexQueueTest()
-    {
-    }
-
-    ~IndexQueueTest()
-    {
-    }
+    IndexQueueTest() = default;
 
-    void SetUp()
-    {
-    }
-
-    void TearDown()
-    {
-    }
+    ~IndexQueueTest() override = default;
```

Comment:
Didn't know that `override` also applies to dtors

Reply:
Apparently. It is in principle also a function and here the base implementation is virtual so clang-tidy complains to be explicit about overriding (even for default).